static/csrc/model_container.cpp

```diff
@@ -403,9 +403,9 @@ void ModelContainer::SetConstantImpl(
           ". Check that the provided tensor's shape is correct.");
     }
   } else {
-    throw std::runtime_error(
```

Comment:
Why this change? We want to throw an error in this case?

Reply:
Maybe there are some additional variables won't be loaded to the ait model, only warning them is a better solution.

Reply:
Do you have any examples? This error means that a constant is neither bound nor unbound, which is unexpected. It means that there is something wrong in this constant.

Reply:
reverted